examples/ring_app/README.md

```diff
@@ -0,0 +1,152 @@
+# Simple Ring App with Authorization Example
+
+This example demonstrates how to incorporate the trapperkeeper-authorization
+service into a simple Ring app.  This is based loosely upon the
+[ring_app example] (https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/tree/master/examples/ring_app)
+in the trapperkeeper-webservices-jetty9 project.  See that example for more
+information on the use of the `jetty9-service` and the Jetty web server
+integration with Ring.
+
+This example includes a simple set of default authorization rules which can be
+used to protect access to the `/hello` endpoint that is handled by the service.
+
+All code needed to execute this example is located in `./src/examples/ring_app`.
+The Clojure code is contained in the `ring_app.clj` file.
+
+## Launching trapperkeeper and running the app
+
+To start up _trapperkeeper_ and launch the sample application, use the
+following _lein_ command while in the _trapperkeeper-authorization_ home
+directory:
+
+~~~~sh
+lein trampoline run --config ./examples/ring_app/ring-example.conf \
+                    --bootstrap-config ./examples/ring_app/bootstrap.cfg
```

Comment:
Considering that we don't expect any of this change, and that we can do it in the dev profile, I think we might as well add a lein alias that does, right?